demos/python_demos/object_detection_retinaface_demo/detector.py

```diff
@@ -0,0 +1,67 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import cv2
+import os
+import numpy as np
+
+from postprocessor import FacialLandmarksPostprocessor
+
+class Detector(object):
+    def __init__(self, ie, model_path, face_prob_threshold, mask_prob_threshold, device='CPU'):
+        model = ie.read_network(model_path, os.path.splitext(model_path)[0] + '.bin')
+
+        assert len(model.inputs) == 1, "Expected 1 input blob"
+        assert len(model.outputs) == 12 or len(model.outputs) == 9, "Expected 12 or 9 output blobs"
+
+        self._input_layer_name = next(iter(model.inputs))
+        self._output_layer_names = model.outputs
+        _, channels, self.input_height, self.input_width = model.inputs[self._input_layer_name].shape
+        assert channels == 3, "Expected 3-channel input"
+
+        self._detect_masks = True if len(model.outputs) == 12 else False
+        self.face_prob_threshold = face_prob_threshold
+        self.mask_prob_threshold = mask_prob_threshold
```

Comment:
mask_prob_threshold used only  in object_detection_retinaface_demo.py

Reply:
Done